app/src/main/java/com/example/whereto/fragments/PlanTripFragment.java

```diff
@@ -1,21 +1,228 @@
 package com.example.whereto.fragments;
 
+import android.app.DatePickerDialog;
+import android.app.Dialog;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+import com.daprlabs.cardstack.SwipeDeck;
 
+import com.example.whereto.Question;
+import com.example.whereto.QuestionsAdapter;
 import com.example.whereto.R;
+import com.google.android.material.slider.LabelFormatter;
+import com.google.android.material.slider.Slider;
+
+import java.text.NumberFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Currency;
 
 public class PlanTripFragment extends Fragment {
-    public PlanTripFragment() {
+    private SwipeDeck cardStack;
+    private ArrayList<Question> questionList;
+    EditText startDate;
+    EditText endDate;
+    DatePickerDialog datePickerDialog;
+
+    public float budget = 0;
+    public float radius = 0;
+    public String tripStart;
+    public String tripEnd;
+    public boolean food = false;
+    public boolean hotels = false;
+    public boolean tours = false;
+    public boolean athletic = false;
+    public boolean arts = false;
+    public boolean shopping = false;
+    public boolean nightlife = false;
+    public boolean beauty = false;
+
+    public PlanTripFragment() {}
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        getQuestionDeck(view);
+        getTripStartDate();
+        getTripEndDate();
+        getBudget();
+
+        Button planTripButton;
+        planTripButton = view.findViewById(R.id.planButton);
+
+        planTripButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                planTrip();
+            }
+        });
     }
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         return inflater.inflate(R.layout.fragment_plan, container, false);
     }
+
+    public void getQuestionDeck(final View view) {
+        questionList = new ArrayList<>();
+        cardStack = (SwipeDeck) view.findViewById(R.id.swipe_deck);
+
+        questionList.add(new Question("food/restaurants"));
```

Comment:
Could these strings be put into an array and then using a while-loop loaded into this questionList? I would think that an array would be easier to manage and scalable.